model/encoding/signing_tags.go

```diff
@@ -12,13 +8,16 @@ import (
 // a domain tag that specifies the type of the signed object.
 
 func tag(domain string) string {
-	return protocolPrefix + domain + crypto.BLSCipherSuite
```

Comment:
Why was this removed? Is this backwards-compatible change?

Reply:
yes, it was moved here: https://github.com/onflow/flow-go/blob/d5f70f2e8e7b8f8403cc494a3a67381b1edc2f85/crypto/bls.go#L92 to be part of the crypto code and the audit scope.

Reply:
Actually, I just updated it: 
https://github.com/onflow/flow-go/blob/3d8787cc9e8dd60faa689797e9903cee47da8f65/crypto/bls.go#L92